packages/autocomplete-core/src/__tests__/reshape.test.ts

```diff
@@ -0,0 +1,213 @@
+import { createPlayground, runAllMicroTasks } from '../../../../test/utils';
+import { createAutocomplete } from '../createAutocomplete';
+import { AutocompleteReshapeSource } from '../types';
+
+const recentSearchesPlugin = {
+  getSources() {
+    return [
+      {
+        sourceId: 'recentSearchesPlugin',
+        getItems() {
+          return [
+            { label: 'macbook' },
+            { label: 'macbook pro' },
+            { label: 'iphone' },
+          ];
+        },
+      },
+    ];
+  },
+};
+
+const querySuggestionsPlugin = {
+  getSources() {
+    return [
+      {
+        sourceId: 'querySuggestionsPlugin',
+        getItems() {
+          return [
+            { query: 'macbook' },
+            { query: 'macbook air' },
+            { query: 'macbook pro' },
```

Comment:
```suggestion
            { query: 'iPhone' },
            { query: 'iPhone Pro' },
            { query: 'iPhone Pro Max' },
```

Reply:
(need to update the tests for this to work of course, or could be other queries)

Reply:
Updated in 12ae2dbdc49e963a1a578329dc660b60e2aca729.